src/main/webapp/script.js

```diff
@@ -1,3 +1,10 @@
+/**
+ * Redirect to Main Hub Page.
+ */
+function projectTest() {
+  fetch('my-project-test');
```

Comment:
You can also just navigate to `site.com/my-project-test` to see your servlet's output without having to hook it up to your pages.